### PR TITLE
#113 Add saves file to list of local saves on clients as well.

### PR DESCRIPTION
**Pull request by zuev93**

### CODE REVIEW

File: src/MultiplayerMod/Multiplayer/World/WorldManager.cs [hunk 1]
```diff
@@ -31,23 +31,28 @@ public class WorldManager {
...
         LoadScreen.DoLoad(path);
     }
 
-    private static byte[] GetWorldSave() {
-        var path = SaveLoader.GetActiveSaveFilePath();
-        Execution.RunUsingLevel(ExecutionLevel.Multiplayer, () => SaveLoader.Instance.Save(path));
-        return File.ReadAllBytes(path);
+    private static string WorldName => Path.GetFileNameWithoutExtension(SaveLoader.GetActiveSaveFilePath());
+
+    private static byte[] WorldSave {
```
Comment (polycone) [inline src/MultiplayerMod/Multiplayer/World/WorldManager.cs:50]: Are you sure it's a good idea to make it a property?🤔
I would expect it's cached or something. Maybe it's better to create a class, say `WorldSave` with `Name` and `Data` and just pass it to the command?

**Review by polycone — APPROVED** [review]

**Comment by polycone** [comment]
Could you please add cloud / local preference support? (`SaveLoader.GetCloudSavesDefault`)

File: src/MultiplayerMod/Multiplayer/World/WorldManager.cs (hunk 1 above)
Comment (zuev93) [reply src/MultiplayerMod/Multiplayer/World/WorldManager.cs:50]: I don't mind to revert it back.

In general I'm doubting any parametrless method starts with Get in C#. Those feels to be either properties or should be renamed to something else (e.g. Calculate...)

**Comment by zuev93** [comment]
> Could you please add cloud / local preference support? (`SaveLoader.GetCloudSavesDefault`)

Done

**Pull request merged**